<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiTopBar</class>
 <widget class="QWidget" name="GuiTopBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>34</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>34</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="playFromStartButton">
       <property name="toolTip">
        <string>Start playing music from the start</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/play-from-start.png</normaloff>:/images/play-from-start.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="toolTip">
        <string>Start and stop playing music</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="speedSpin"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="keyCombo"/>
     </item>
     <item>
      <widget class="QComboBox" name="majorCombo"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Transpose:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="transposeSpin"/>
     </item>
     <item>
      <widget class="QToolButton" name="loopingBarsPopupButton">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::DownArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start Bar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="startBarSpin">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>999.899999999999977</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBarButton">
       <property name="toolTip">
        <string>Save this Bar Number</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/flag.png</normaloff>:/images/flag.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="themeCobox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
